docs/en/ingest-management/commands.asciidoc

```diff
@@ -227,14 +229,18 @@ specified {es} URL.
 `--fleet-server-es-ca <string>`::
 Path to certificate authority to use to communicate with {es}.
 
+`--fleet-server-es-ca-trusted-fingerprint <string>`::
+The hash of {es}'s certificate authority that will used by
```

Comment:
Sentence structure here makes the details a bit harder to follow. Also it's not clear to me where the user gets the fingerprint. Does elasticsearch-certutil spit it out when you generate self-signed certs? Will users know how to get this hash? Are there related ES docs we should be pointing to?

I'll suggest an edit here with the caveat that I'm not sure I have all the technical details quite right. 

```
The SHA256 fingerprint (hash) of the certificate authority used to self-sign {es} certificates.
This fingerprint will be used to verify self-signed certificates presented by {fleet-server} and
any inputs started by {agent} for communication. This flag is required when using self-signed
certificates with {es}.
```

Another questions: is this related to  `--ca-sha256`?

Reply:
@dedemorton, your description looks correct. The only change I'd made is to add a hyphen for `SHA-256`. 

I have a related PR open for [connecting clients to Elasticsearch](https://github.com/elastic/elasticsearch/pull/81279) in 8.0+. When you start Elasticsearch for the first time, a CA certificate is generated and stored on disk, and the hex-coded SHA-256 fingerprint of this certificate is also output to the terminal. In the terminal will be printed something like this:

```
The hex-encoded SHA-256 fingerprint of the generated HTTPS CA DER-encoded certificate:
b9a10bbe64ee9826abeda6546fc988c8bf798b41957c33d05db736716513dc9c
```

This is the fingerprint of the CA certificate stored on disk, which is named `http_ca.crt`. Here's a list of the [security certificates](https://www.elastic.co/guide/en/elasticsearch/reference/8.0/configuring-stack-security.html#stack-security-certificates) that are created and stored on disk.

Reply:
@michel-laterman Maybe we should all more detail about where users find this file? Or point to the security docs.